### ISSUE TITLE
[Serve] [Docs] misspell in User Guides

### ISSUE THREAD

**Issue by kimikuri** [ray-project/ray#25397]
### Description

Hi, when I checked the official ray serve docs about "User Guides", I found the misspell shown as the screen shot at Link. 
Can I send PR?

### Link

[User Guides](https://docs.ray.io/en/master/serve/user-guide.html)
![Screenshot](https://user-images.githubusercontent.com/87696872/171583839-b4ea7603-eefe-4a7b-8719-1f71ff7751d9.png)

**Comment by tupui** [comment]
Hi @kimikuri, thank you for reporting. PR improving documentation are always welcomed. It would be better though if you could group this with other typo fixes as any PR requires minimal work from maintainers and has a cost in terms of CI. Hence we try to limit these.

**Comment by kimikuri** [comment]
Hi @tupui, Thank you for your reply.
I understood the manner for fixing typo(really appreciate!), but I'm sorry, now I found only this.
Then which is better (1) I send PR or (2) no PR (only reporting in this issue)?
I follow you.

**Comment by tupui** [comment]
Better IMO is then not to make a PR until you have more typos.

**Comment by kimikuri** [comment]
OK! Thanks a lot!

**Comment by zhe-thoughts** [comment]
Yup this is a good general guideline @tupui (thank you for helping with new community members!)

@kimikuri : I would say that if you find more typos or small fixes *on the same page*, please batch them into a PR. Otherwise, if you already have finished reading the page and this is the only small fix you find, filing a PR is fine. (We have recently optimized our CI so that docs only changes are much easier to verify)

Basically a "sliding window grouping" 😆

**Comment by simon-mo** [comment]
Please feel free to tag me (@simon-mo) on that PR, I made that type 😅

**Comment by kimikuri** [comment]
Thanks to @zhe-thoughts and @simon-mo! I've sent the PR after checking that page again and there are no  typo.
Adding to that, I checked the similar typos "tunning" in all doc, and there were nothing but here.
